src/main/scala/firrtl/passes/InferWidths.scala

```diff
@@ -41,13 +41,13 @@ object InferWidths extends Pass {
       case wx => wx
     }
     def collectMinMax(w: Width): Width = w map collectMinMax match {
-      case MinWidth(args) => MinWidth(unique((args.foldLeft(Seq[Width]())) {
-        case (res, wxx: MinWidth) => res ++ wxx.args
-        case (res, wxx) => res :+ wxx
+      case MinWidth(args) => MinWidth(unique((args.foldLeft(List[Width]())) {
+        case (res, wxx: MinWidth) => wxx.args.toList ++ res
```

Comment:
I'm actually not certain if this matters in practice (eg. compile *may* do the right thing automatically, but you could do `wxx.args ++: res` so that the append operator returns the type of the right collection rather than the left collection like `++` does. This potentially saves a traversal and construction of a List from the args of `wxx`

Reply:
Done.